package-lock.json

```diff
@@ -2,6 +2,16 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
+    "JSONStream": {
```

Comment:
All of these changes seem to just reorder things, so I'm not sure this is needed in this PR?